core/modules/tests.rs

```diff
@@ -275,7 +288,8 @@ impl ModuleLoader for MockLoader {
 
   fn code_cache_ready(
     &self,
-    module_specifier: &ModuleSpecifier,
+    module_specifier: ModuleSpecifier,
```

Comment:
Changed this to an owned value because deno_core has it available without cloning and this can help prevent cloning if someone wants to send this to another thread when saving the code cache.